src/sniffer/CmdHandler.c

```diff
@@ -30,6 +43,29 @@ void CmdHandler_PrintCommandList() {
 
     printf("-= %d =- [IDLE] Do nothing...\n", (int) CmdHandler_cmdCode_IDLE);
     printf("-= %d =- [EXIT] Safely exits the program.\n", (int) CmdHandler_cmdCode_EXIT);
+    printf("-= %d =- [SAVE] Save the log to a .txt file.\n", (int) CmdHandler_cmdCode_SAVE);
 
     printf("\n");
 }
+
+int CmdHandler_SaveLogAsTxt(char* fileDest) {
```

Comment:
Why not just move the file and rename it?

Reply:
You won't be able to save more than once if we do that